src/super_gradients/training/transforms/keypoint_transforms.py

```diff
@@ -0,0 +1,302 @@
+import random
+from abc import abstractmethod
+from typing import Tuple, List, Iterable
+
+import cv2
+import numpy as np
+from torchvision.transforms import functional as F
+
+__all__ = [
```

Comment:
I think it would be a good idea to add tests with fixed key points expected results for the below transforms.

Reply:
Done